lib/ui/painting.dart

```diff
@@ -3007,14 +3007,9 @@ class ColorFilter {
   static const int _TypeLinearToSrgbGamma = 3; // MakeLinearToSRGBGamma
   static const int _TypeSrgbToLinearGamma = 4; // MakeSRGBToLinearGamma
```

Comment:
these should be prefixed with a `k` to match our style guide (`_kTypeMode`, etc). Right now they look like Dart classes...